gui/wxpython/rdigit/g.gui.rdigit.html

```diff
@@ -0,0 +1,86 @@
+<!-- meta page description: wxGUI Raster Digitizer -->
```

Comment:
This is a copy of wxpython/docs/wxGUI.rdigit.html, I think there is a mechanism so that it doesn't have to be duplicated, see vdigit, it's not in docs, but the website wxGUI.vdigit.html still exists. So I would try to remove the one in docs and see if that breaks anything.

Reply:
Yes, I already fixed it.